http/1.5/modules/ROOT/pages/http-start-app-brows-task.adoc

```diff
@@ -1,29 +1,84 @@
-= Start an App from a Browser
+= Start an App from a Browser Example - Mule 4
 :page-aliases: connectors::http/http-start-app-brows-task.adoc
 
-In this procedure, you configure an HTTP Listener to specify a host name, port, and path. To listen on a specific port for every network interface on the server, set the host to `0.0.0.0`. Using `localhost` listens for incoming requests generated inside the server. Start a browser and submit a URL. The result appears in the Anypoint Studio Console.
+The following example illustrates how to configure Anypoint Connector for HTTP (HTTP Connector) *Listener* source to initiate a flow by starting an app from a browser and listen on a specific port for every network interface on the server. Then, the *Logger* component logs the message `Hello`.
```

Comment:
Added "the", split the sentences to make them shorter,  and made the wording parallel. You may want to rephrase this, but try to keep the wording parallel (that is, "listen" and "log" or "listens" and "logs").

The following example illustrates how to configure the Anypoint Connector for HTTP (HTTP Connector) *Listener* source to initiate a flow. In this example, HTTP Listener listens on a specific port for every network interface on the server and starts the app when it hears ???.  Then the *Logger* component logs the message "Hello."